helm/templates/service.yaml

```diff
@@ -8,8 +8,6 @@ spec:
   type: {{ .Values.service.type }}
   ports:
     - port: {{ .Values.service.port }}
-      targetPort: http
```

Comment:
why remove `targetPort`? where will service forward request to?

Reply:
reverting this back.